src/main/java/com/google/graphgeckos/dashboard/storage/Builder.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+
+/**
+ * Contains the information retrieved from a single build bot. It is used as a subclass
+ * of {@link #BuildInfo BuildInfo}, and for converting ParsedBuildbotData objects
+ * to Builder objects, used by BuildInfo.
+ */
+@Entity(name = "builder")
+public class Builder {
+
+  private final String builderName;
```

Comment:
Consider calling this just "name" (and update related members, getters, setters, accordingly): it's implied that it is the name of this builder object.